app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,17 @@
 class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+
+    @comment = Comment.new(user: @current_user, post: @post, text: params[:text])
```

Comment:
- [ ] [OPTIONAL] For creating any model attributes, always use strong parameters, this increases the security of data sent using forms.